fe/src/components/UI/molecules/cardItem/OPCardItem.tsx

```diff
@@ -17,17 +19,17 @@ const OPCardItem: React.FC<IOPCardItemProps> = ({
   status,
   shortDescription,
   imageUrl,
+  onClickEdit,
+  item
 }) => {
   const color: string = getRandomColor(id);
 
   const style = {
     backgroundColor: color,
   };
 
-  console.log(imageUrl);
-
   return (
-    <div className={classes["card-item-wrapper"]} style={imageUrl && imageUrl.length > 0 ? {} : {paddingTop: "42px", minHeight: "261px"}}>
+    <div onClick={() => {onClickEdit(item)}} className={classes["card-item-wrapper"]} style={imageUrl && imageUrl.length > 0 ? {} : {paddingTop: "42px", minHeight: "261px"}}>
```

Comment:
Don't add inline styles for paddingTop: "42px", minHeight: "261px". Better use dynamic classes